core-java/src/main/java/com/baeldung/extension/Extension.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.extension;
+
+import com.google.common.io.Files;
+import org.apache.commons.io.FilenameUtils;
+
+public class Extension {
+    //Instead of file name we can also specify full path of a file eg. /baeldung/com/demo/abc.java
+    public String getExtensionByApacheCommonLib(String fileName) {
```

Comment:
please update the argument name "fileName" to be "filename" to match the rest of the file.

Reply:
Done. I have changed the other names because it violates the camel case convention.

Reply:
Camel Case only counts for multiple words. ["filename"]](http://www.dictionary.com/browse/filename) is a single word, not two.